test/test_dockerpy/README.md

```diff
@@ -2,29 +2,43 @@
 
 Python test suite to validate Podman endpoints using dockerpy library
 
+Validate :-
+
+Make sure podman binary exist under libpod/bin
```

Comment:
```suggestion
Make sure the Podman binary exists under libpod/bin
```

Reply:
although on further thought, don't uppercase Podman there, you're searching for the binary 'podman', not the binary 'Podman'.  Keep the and exists though